styles.css

```diff
@@ -92,6 +93,206 @@ h2 {
   justify-content: center;
 }
 
+.grid-container {
+  display: grid;
+  grid-column: auto;
+  grid-row: auto;
+  gap: 24px;
+  place-items: center;
+}
+
+.work h2 {
+  width: 327px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  align-items: center;
+  margin: 0 auto 24px auto;
+}
+
+.indicator {
+  align-items: center;
+  border-radius: 24px;
+  background: #36b37e;
+  width: 48px;
+  height: 4px;
+  margin: 0 auto 60px auto;
+}
+
+.work-1,
+.work-2,
+.work-3,
+.work-4,
+.work-5,
+.work-6 {
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+  width: 327px;
+  height: 474px;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  border-radius: 8px;
+  width: 325px;
+  height: 222px;
+  padding: 16px;
+  text-align: center;
+}
+
+.project h3 {
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.prog-lan {
+  align-items: baseline;
+  display: flex;
+  column-gap: 5px;
+  list-style: none;
+}
+
+.prog-lan li {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: center;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.btn {
+  width: auto;
+  padding: 12px;
+  background: #36b37f;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border-style: none;
+  letter-spacing: 0.03em;
+}
+
+.resume {
+  display: flex;
+  flex-direction: column;
+}
+
+.resume h2 {
+  position: absolute;
+  right: 20px;
+  width: 327px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  align-items: center;
+  margin: 50px auto 25px auto;
+}
```

Comment:
I am really impressed with your project so far. It pains me not to approve this project, but since it is part of the project requirements you have to work on it and submit for review again. You added right: 20px to the resume h2 on line 203 which causes it to push to the right as shown below. Please check and rectify this.  

![Screenshot from 2022-09-23 17-33-24](https://user-images.githubusercontent.com/65233860/192024777-62e09784-b038-4821-b1af-365d4c2c2b6e.png)